lib/utils/version.js

```diff
@@ -115,11 +115,14 @@ const utils = {
             return null;
         }
 
-        if (activeMajor === 1 && activeVersion !== latestV1 && semver.diff(activeVersion, versionToUse) === 'major') {
+        const latestMajor = latestMajorVersions[`v${activeMajor}`];
+
+        if (activeVersion !== latestMajor && semver.diff(activeVersion, versionToUse) === 'major') {
             const majorToUse = semver.major(versionToUse);
+
             throw new CliError({
-                message: `You are trying to update to Ghost v${majorToUse}, but your blog is not on the latest Ghost 1.0 version`,
-                help: 'Instead run "ghost update v1".'
+                message: `You are trying to update to Ghost v${majorToUse}, but your blog is not on the latest Ghost ${activeMajor}.0 version`,
+                help: `First upgrade to the latest Ghost v${activeMajor} with "ghost update v${activeMajor}", then update to Ghost ${majorToUse} with "ghost update".`
```

Comment:
I would nuke the first word personally:

```suggestion
                help: `Upgrade to the latest Ghost v${activeMajor} with "ghost update v${activeMajor}", then update to Ghost ${majorToUse} with "ghost update".`
```

Reply:
done